### ISSUE TITLE
Need automated process for checking links.

### ISSUE THREAD

**Issue by etsauer** [redhat-cop/uncontained.io#18]
@JaredBurck @rdebeasi is there a native hugo way to do this? otherwise, we already have a Gemfile in order to install asciidoctor, we could pull in htmlchecker as well.

**Comment by rdebeasi** [comment]
I'm not aware of a "native Hugo" way to do this. The author of Hugo considers a link checker to be [out of scope](https://github.com/gohugoio/hugo/issues/1430). Some folks who want functionality like this set up a build tool/package manager such as npm around their Hugo site - see the [starter kits](https://gohugo.io/tools/starter-kits/) for some examples. In our case, I imagine that build tool is probably Docker.

**tl;dr:** Yeah, htmlchecker sounds great to me! :smile:

**Comment by etsauer** [comment]
@rdebeasi @JaredBurck looking to tackle this by adding a "test" directive to our gulpfile, and using https://www.npmjs.com/package/broken-link-checker

**Comment by etsauer** [comment]
#103 resolved this issue.
